src/Resources/views/CRUD/Association/edit_many_script.html.twig

```diff
@@ -462,6 +462,14 @@ This code manages the many-to-[one|many] association field popup
 
             Admin.log('[{{ id }}|on:change] update the label');
 
+            // Skip short object information call if object is missing.
+            if (!jQuery(this).val()) {
```

Comment:
I feel like there is some refactoring that we can do.

- `jQuery(this).val()` is used multiple times, and could be set inside a variable instead.
- `jQuery('#field_actions_{{ id }} a.btn-warning').addClass('hidden');` is used two times and the second times seems useless now
- There is a `{% if btn_edit %}` check you didn't used.

Something like this could be an improvement:
```
var objectId =  jQuery(this).val();
if (!objectId) {
      jQuery('#field_widget_{{ id }}').html('{{ 'short_object_description_placeholder'|trans({}, 'SonataAdminBundle') }}');

     {% if btn_edit %}
         jQuery('#field_actions_{{ id }} a.btn-warning').addClass('hidden');
     {% endif %}

     return;
}

jQuery('#field_widget_{{ id }}').html("<span><img src=\"{{ asset('bundles/sonataadmin/ajax-loader.gif') }}\" style=\"vertical-align: middle; margin-right: 10px\"/>{{ 'loading_information'|trans([], 'SonataAdminBundle') }}</span>");
jQuery.ajax({ ... }

{% if btn_edit %}
    var edit_button_url = '{{
        associationadmin.generateUrl('edit', {'id' : 'OBJECT_ID'})
    }}'.replace('OBJECT_ID', objectId);

    jQuery('#field_actions_{{ id }} a.btn-warning').removeClass('hidden').attr('href', edit_button_url);
{% endif %}
```

Reply:
Thanks for the suggestion, I have apply your changes.